core/src/main/java/org/elasticsearch/index/query/HasChildQueryBuilder.java

```diff
@@ -84,11 +119,35 @@ public HasChildQueryBuilder setShortCircuitCutoff(int shortCircuitCutoff) {
     /**
      * Sets inner hit definition in the scope of this query and reusing the defined type and query.
      */
-    public HasChildQueryBuilder innerHit(QueryInnerHitBuilder innerHit) {
-        this.innerHit = innerHit;
+    public HasChildQueryBuilder innerHit(QueryInnerHits queryInnerHits) {
+        this.queryInnerHits = queryInnerHits;
         return this;
     }
 
+    public QueryBuilder getInnerQueryBuilder() {
+        return queryBuilder;
+    }
+
+    public String getChildType() {
+        return childType;
+    }
+
+    public ScoreType getScoreType() {
+        return scoreType;
+    }
+
+    public int getMinChildren() {
+        return minChildren == null ? 0 : minChildren;
+    }
+
+    public int getMaxChildren() {
+        return maxChildren == null ? 0 : maxChildren;
+    }
+
+    public int getShortCircuitCutoff() {
+        return shortCircuitCutoff == null ? 8192 : shortCircuitCutoff;
```

Comment:
same should be done for minChildren and maxChildren too. Wondering now if it makes sense to move to `int` rather than `Integer` for all three fields